wger/core/views/user.py

```diff
@@ -246,7 +246,7 @@ def registration(request):
     template_data['submit_text'] = _('Register')
     template_data['extend_template'] = 'base.html'
 
-    return render(request, 'form.html', template_data)
+    return render(request, 'reg_form.html', template_data)
```

Comment:
I had thought about that but there seemed to be other pages extending from form.html so I thought I could create a form specifically for the registration page